module/zfs/arc.c

```diff
@@ -600,16 +600,15 @@ arc_stats_t arc_stats = {
 	{ "abd_chunk_waste_size",	KSTAT_DATA_UINT64 },
 };
 
+arc_wmsums_t arc_wmsums;
```

Comment:
Some of the elements of this structure are not of type wmsum_t
- some of the elements are of type aggsum_t
- perhaps we could use arc_sums as a more generic name?